react/features/stream-effects/virtual-background/JitsiStreamBackgroundEffect.js

```diff
@@ -83,7 +82,7 @@ export default class JitsiStreamBackgroundEffect {
         //
 
         // Smooth out the edges.
-        if (this._options.virtualBackground.isVirtualBackground) {
+        if (this._options.virtualBackground.backgroundEffectEnabled) {
```

Comment:
This is incorrect. We'll apply a 4px edge smoothing when using an image and an 8px one when blurring the whole thing.

Reply:
Changed by background type.